src/marketing/recommendation-change-source.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Fetch Product Recommendations from a Different Source
+tag: product-recommendations
+ee_only: true
+---
+
+A Magento store configured to use product recommendations fetches those recommendations through a [SaaS service]({% link configuration/services/saas.md %}) linked to your production environment.
```

Comment:
Needs context along the lines of: 

When you configure Recommendations with a SaaS Env, that SaaS Env collects catalog data and storefront behavioral data. Adobe Sensei analyzes this data collected and computes product associations used to serve Product Recommendations. 

The challenge is that on non-prod envs like testing or staging, there is often not much storefront data (unless automated) to feed Recommendations. Real shopper behavior at scale is captured only on the production env. 

Magento allows you to use the "learnings" based on shopper events in your production environment on other, non-prod envs. This allows you to preview which recommendations your shoppers will see and experiment with different recommendation types and placement locations.

Through the Admin UI in your testing environment, you can fetch recommendations generated from actual shopper data from your production environment or another environment. Changing where those recommendations are fetched is done by specifying a different SaaS environment.

Requirements: 
1. Storefront data collection is configured and enabled on production
2. Catalog on your non-prod envs is largely the same as you have on production. Staging is a good candidate because it's a best practice to have your catalog there be analogous to your production catalog. 

Important: when recommendations from another saas env are displayed on storefront, they will not be clickable. They are served for preview purposes only.